cdap-watchdog/pom.xml

```diff
@@ -70,6 +70,27 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-common</artifactId>
+      <scope>provided</scope>
```

Comment:
Without this, you'll see compilation errors like:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project cdap-watchdog: Compilation failure: Compilation failure:
[ERROR] /home/gary/devel/cdap/cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java:[47,31] package org.apache.hadoop.hbase does not exist
[ERROR] /home/gary/devel/cdap/cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java:[48,38] package org.apache.hadoop.hbase.client does not exist
[ERROR] /home/gary/devel/cdap/cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java:[184,7] cannot find symbol
[ERROR] symbol:   class HBaseAdmin
[ERROR] location: class co.cask.cdap.metrics.store.upgrade.MetricsDataMigrator
[ERROR] /home/gary/devel/cdap/cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java:[184,32] cannot find symbol
[ERROR] symbol:   class HBaseAdmin
[ERROR] location: class co.cask.cdap.metrics.store.upgrade.MetricsDataMigrator
[ERROR] /home/gary/devel/cdap/cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java:[185,12] cannot find symbol
[ERROR] symbol:   class HTableDescriptor